tests/test_algorithm.py

```diff
@@ -3665,12 +3666,18 @@ def test_daily_delisted_equities(self,
         transactions = output['transactions']
         initial_fills = transactions.iloc[1]
         self.assertEqual(len(initial_fills), len(assets))
+
+        last_minute_of_session = \
+            self.trading_calendar.open_and_close_for_session(
+                self.test_days[1]
+            )[1]
```

Comment:
I've noticed that we often want just the open or just the close, and we end up calling this to get both and then ditching one.  Maybe we need `get_session_open` and `get_session_close`?